api-reference/_hidden/dxGanttHeaderFilterTexts/cancel.md

```diff
@@ -5,7 +5,6 @@ default: 'Cancel'
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies text for the "Cancel" button in the popup window.
```

Comment:
By "text" do you mean the button's caption?

Reply:
Yes.